i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-trendmicro-iwsva-snmp.md

```diff
@@ -2,48 +2,636 @@
+| CRITICALIN     | Critical thresholds for in traffic                                                                                                                                                                                  | 90                |             |
+| WARNINGOUT     | Warning thresholds for out traffic                                                                                                                                                                                  | 80                |             |
+| CRITICALOUT    | Critical thresholds for out traffic                                                                                                                                                                                 | 90                |             |
+| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%{admstatus} eq "up" and %{opstatus} ne "up"'). You can use the following variables: %{admstatus}, %{opstatus}, %{duplexstatus}, %{display} |                   |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                    | --verbose         |             |
+
+</TabItem>
+</Tabs>
+
+3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.
+
+## Comment puis-je tester le plugin et que signifient les options des commandes ?
+
+Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
+de commande depuis votre collecteur Centreon en vous connectant avec
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_trendmicro_iwsva.pl \
+	--plugin=os::linux::snmp::plugin \
+	--mode=interfaces \
+	--hostname=10.0.0.1 \
+	--snmp-version='2c' \
+	--snmp-community='my-snmp-community'  \
+	--interface='.*' \
+	--name \
+	--add-status \
+	--add-traffic \
+	--critical-status='' \
+	--warning-in-traffic='80' \
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
 
-  - Read-Only SNMP community
-  - IP Address of the equipment
+```bash
+OK: All interfaces are ok | '*interface_name*#status'=;;;;'*interface_name*#interface.traffic.in.bitspersecond'=b/s;;;;'*interface_name*#interface.traffic.out.bitspersecond'=b/s;;;;
```

Comment:
no values?

Reply:
Was surprise but [Linux-SNMP](https://docs.centreon.com/fr/pp/integrations/plugin-packs/procedures/operatingsystems-linux-snmp/#comment-puis-je-tester-le-plugin-et-que-signifient-les-options-des-commandes-) has the same so I supposed it was ok.